### ISSUE TITLE
JRE crash with zero-thickness mesh shape in Minie v4.5

### ISSUE THREAD

**Issue by stephengold** [stephengold/Minie#19]
Reported by Koteinik at the JMonkeyEngine Forum/Hub:  https://hub.jmonkeyengine.org/t/fatal-error-in-jre-when-using-minie/45129

This appears to be a regression in Minie v4.5 caused by the fix to issue #18.

Here's a test that triggers the crash with a debug native library:
```java
    public void simpleInitApp() {
        BulletAppState bulletAppState = new BulletAppState();
        stateManager.attach(bulletAppState);

        Node playerNode = new Node();
        rootNode.attachChild(playerNode);

        BetterCharacterControl player
                = new BetterCharacterControl(0.5f, 6f, 2f);
        playerNode.addControl(player);
        bulletAppState.getPhysicsSpace().add(player);

        Box b = new Box(0f, 1f, 16f);
        CollisionShape shape = new MeshCollisionShape(b);
        PhysicsRigidBody phys
                = new PhysicsRigidBody(shape, PhysicsBody.massForStatic);
        bulletAppState.getPhysicsSpace().addCollisionObject(phys);
    }
```

**Comment by stephengold** [comment]
The fix is taking longer than expected, mainly due to a build issue at Travis CI:  ticket 34567.

**Comment by stephengold** [comment]
Typical console output:
```text
Debug_Libbulletjme version 12.5.0 initializing
java: /home/travis/build/stephengold/Libbulletjme/src/main/native/bullet3/LinearMath/btVector3.h:305: btVector3& btVector3::normalize(): Assertion `!fuzzyZero()' failed.
```

**Comment by stephengold** [comment]
The fix is in Libbulletjme v12.6.0, which was just released today.

**Comment by stephengold** [comment]
This fix is in Minie v4.6.0.
